### ISSUE TITLE
XELB process objects accumulate constantly, even if X client windows killed

### ISSUE THREAD

**Issue by wgreenhouse** [ch11ng/exwm#102]
Steps to reproduce:

1) Install and run exwm 0.1 from GNU ELPA.

2) Open and close X client windows.

3) Keep EXWM running for a prolonged period, e.g.

```
(emacs-uptime)
```
1 day, 3 hours, 5 minutes, 47 seconds

4) Note that the number of XELB process objects bears no relation to the number of EXWM buffers.

```emacs-lisp
(length (remove-if-not (lambda (buf) (with-current-buffer buf (eq major-mode 'exwm-mode))) (buffer-list)))
```
1

```emacs-lisp
(length (remove-if-not (lambda (proc) (string-match "XELB" (process-name proc))) (process-list)))
```
31

This does not appear to have any stability consequences that I can see yet; in fact I've found EXWM very stable so far. But I wonder if it hints at a memory leak or failure to garbage collect that may bite me in the future.

**Comment by ch11ng** [comment]
Did you use EXIM? If so then the problem should be fixed in ch11ng/exim@e61547a4eea6f0496456c391cff12711342c2660.

**Comment by wgreenhouse** [comment]
Chris Feng <notifications@github.com> writes:

> Did you use EXIM? If so then the problem should be fixed in ch11ng/exim@e61547a4eea6f0496456c391cff12711342c2660.

Thank you; this did resolve the issue.
